src/bot.py

```diff
@@ -41,19 +50,28 @@ def main():
     dispatcher.add_handler(CommandHandler(DINNER, handle_menu(meal=DINNER)))
     dispatcher.add_handler(CommandHandler(SETTINGS, handle_settings))
     dispatcher.add_handler(CommandHandler(HIDE_CUISINE, handle_hidden_cuisine))
+    dispatcher.add_handler(CommandHandler(HIDE_CUISINE, handle_hidden_cuisine))
+    dispatcher.add_handler(CommandHandler(SET_BREAKFAST_NOTIFICATION, handle_subscribe(meal=BREAKFAST)))
+    dispatcher.add_handler(CommandHandler(SET_DINNER_NOTIFICATION, handle_subscribe(meal=DINNER)))
 
     # add callback_query handler
     dispatcher.add_handler(CallbackQueryHandler(handle_start, pattern='^start.+'))
     dispatcher.add_handler(CallbackQueryHandler(handle_settings, pattern='^settings.home'))
     dispatcher.add_handler(CallbackQueryHandler(handle_hidden_cuisine, pattern='^settings.hidden'))
+    dispatcher.add_handler(CallbackQueryHandler(handle_notification, pattern='^settings.notification'))
```

Comment:
actually come to think of it, what if instead of callback queries, we just made the inline buttons send a command instead, so if they click on start, it just makes the user send /start, then we dont need to manage two handlers

Reply:
can it?

Reply:
ok the concern here is that for hide cuisine side, im editing the inlinekeyboardmarkup. Having the user sending text will disrupt the UX.